openmetadata-ui/src/main/resources/ui/src/components/Team/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -886,34 +889,48 @@ const TeamDetailsV1 = ({
   const emailElement = (
     <Space className="m-b-xs">
       {isEmailEdit ? (
-        <Space>
-          <Input
-            className="w-64"
-            data-testid="email-input"
-            placeholder={t('label.enter-entity', {
-              entity: t('label.email-lowercase'),
-            })}
-            type="email"
-            value={email}
-            onChange={(e) => setEmail(e.target.value)}
-          />
-          <Button
-            className="h-8 p-x-xss"
-            data-testid="cancel-edit-email"
-            size="small"
-            type="primary"
-            onClick={() => setIsEmailEdit(false)}>
-            <CloseOutlined />
-          </Button>
-          <Button
-            className="h-8 p-x-xss"
-            data-testid="save-edit-email"
-            size="small"
-            type="primary"
-            onClick={handleUpdateEmail}>
-            <CheckOutlined />
-          </Button>
-        </Space>
+        <Form
+          initialValues={{ email: currentTeam.email }}
+          onFinish={handleUpdateEmail}>
+          <Space align="baseline">
```

Comment:
Not sure why we need Space for form layout

Reply:
@chirag-madlani , we have one input and 2 buttons so to align them we need space.